### ISSUE TITLE
Change auto record time and dates in app. 

### ISSUE THREAD

**Issue by lxlngr** [zipleen/tvheadend-iphone-client#230]
Now when you press ‘auto record’ eg. 911, it automatically records all 911 programs on all days at any time.  
Would love to see it changeable in the app. Now I have to go to he webui to unmark unwanted days of the week abd set anspecific timeframe. 

Hope you know what I mean.

**Comment by zipleen** [comment]
Hi,

This feature already exists for more than 4 or 5 years. Go to recordings, swipe to "autorec", click on the newly created record and you'll be able to swap your options and hit save.

**Comment by lxlngr** [comment]
Oooww. I have to swipe to autorec. FML. 

THANKS!

**Comment by zipleen** [comment]
Yeah it's not very intuitive, but that's why I "move" to autorec he first time a user goes into recordings - unfortunately it's a very sudden move lol

**Comment by lxlngr** [comment]
Sorry but incannot change the different dates. When I tap the days. Nothing happens. Is it me again?

**Comment by zipleen** [comment]
Yeah.. it seems that weekdays is one of the fields that's not codec - it's too custom thus it needs to be codec by hand and I never gave the trouble to implement that one.

There's other fields that don't work as well, like priority. But channel and tag and the text fields do work! Hehe

**Comment by zipleen** [comment]
If you're curious and understand programming, here's the specific piece of code, that hasn't changed for three years or more - https://github.com/zipleen/tvheadend-iphone-client/blob/master/TvhClient/TVHAutoRecDetailViewController.m
